project/src/main/webapp/events-feed.js

```diff
@@ -1,15 +1,20 @@
-$(async function() {
-  const allEvents = await getAllEvents();
-  populateAllEvents(allEvents);
+$(function() {
+  populateEvents();
 });
 
+async function populateEvents() {
+  const rankedEvents = await getRankedEvents();
+  const lodMap = getLodsFromEvents(rankedEvents);
```

Comment:
What's lodMap? Try to use a more meaningful name if possible?

Reply:
Renamed to eventLevels instead, done